examples/launcher/src/components/Dom.jsx

```diff
@@ -229,14 +269,51 @@ class DomDetail extends React.Component {
   /* constructor(props) {
     super(props);
   } */
+  toggleOpen() {
```

Comment:
If [`DomDetail` is removed](https://github.com/exokitxr/exokit/pull/1076/files#diff-43a85fe06ede43ea8d3091dd359ef2b7L102), why is this being edited as well?

Reply:
reverted in 
ad9ecad